cypress/e2e/tests/customMetrics/metrics_application_inventory.test.ts

```diff
@@ -0,0 +1,51 @@
+import {
+    createMultipleApplications,
+    deleteApplicationTableRows,
+    login,
+    preservecookies,
+} from "../../../utils/utils";
+import { Assessment } from "../../models/migration/applicationinventory/assessment";
+import { Metrics } from "../../models/migration/custom-metrics/custom-metrics";
+
+let applicationList: Array<Assessment> = [];
+let metrics = new Metrics();
+
+describe(["@tier2"], "Metrics konveyor_applications_inventoried", function () {
```

Comment:
describe(["@tier2"], "Custom Metrics - The total number of initiated assessments", function () {